src/eko/couplings.py

```diff
@@ -0,0 +1,795 @@
+# -*- coding: utf-8 -*-
+r"""
+This file contains the QCD beta function coefficients and the handling of the running
+coupling :math:`\alpha_s`.
+
+See :doc:`pQCD ingredients </theory/pQCD>`.
+"""
+
+import logging
+
+import numba as nb
+import numpy as np
+import scipy
+
+from . import constants, thresholds
+from .beta import b_qcd, b_qed, beta_qcd, beta_qed
+
+logger = logging.getLogger(__name__)
+
+
+def couplings_mod_ev(mod_ev):
+    """Map ModEv key to the available strong coupling evolution methods"""
+    if mod_ev in ["EXA", "iterate-exact", "decompose-exact", "perturbative-exact"]:
+        return "exact"
+    if mod_ev in [
+        "TRN",
+        "truncated",
+        "ordered-truncated",
+        "EXP",
+        "iterate-expanded",
+        "decompose-expanded",
+        "perturbative-expanded",
+    ]:
+        return "expanded"
+    raise ValueError(f"Unknown evolution mode {mod_ev}")
+
+
+@nb.njit(cache=True)
+def expanded_lo(ref, beta0, lmu):
```

Comment:
Actually, this function should be called `exact_lo` as we can solve the RGE at LO exact and so we do always